types/helmet/helmet-tests.ts

```diff
@@ -70,8 +70,11 @@ function contentSecurityPolicyTest() {
 
     function reportUriCb(req: express.Request, res: express.Response) { return '/some-uri'; }
     function reportOnlyCb(req: express.Request, res: express.Response) { return false; }
-
-    app.use(helmet.contentSecurityPolicy({}));
+    // prettier-ignore
```

Comment:
This `prettier-ignore` doesn't seem needed (though maybe I'm missing something). Other than that, this looks good.

Reply:
I believe it is oversight from non-commited change I've done (limit types in directive keys, was required to not move assertion to different line).
Thanks for spotting this, to be amended

Reply:
done